docs/develop/api_for_vms.rst

```diff
@@ -0,0 +1,46 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
+2. Start Date
+3. Venue
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
+
+In case of a ``GET`` request, a list containing the details of all meetups
+in the ascending order of their dates will be returned.
+
```

Comment:
Provide examples for GET and POST requests.
Each example could include a sample request and the json response from the api